src/sys/select.rs

```diff
@@ -68,11 +87,18 @@ mod ffi {
     }
 }
 
-pub fn select(nfds: c_int,
-              readfds: Option<&mut FdSet>,
+pub fn select(readfds: Option<&mut FdSet>,
```

Comment:
If you're feeling particularly ambitious, this would be a good time to change the definiton of select to something like `pub fn select<T: Into<Option<&mut FdSet>>>(readfds: T ...` such that all the `Option` types have those kinds of signatures. Then you can use `.into()` within the function and the users of the functions don't need to wrap arguments in `Some()`. It's really ergonomic!